test_communication/test/test_publisher_subscriber.cpp

```diff
@@ -33,8 +33,8 @@ void publish(
   auto publisher = node->create_publisher<T>(
     std::string("test_message_") + message_type, custom_qos_profile);
 
-  rclcpp::WallRate time_between_cycles(1);
-  rclcpp::WallRate time_between_messages(10);
+  rclcpp::WallRate time_between_cycles(10);
```

Comment:
Good point. Addressed in a separate commit: 5d2532c50fcdef807cc77e305f23b910e4bf32b2